src/parser/metrics/Classes.ts

```diff
@@ -17,27 +17,59 @@ export class Classes implements Metric {
     private readonly statementsSuperSet: QueryStatementInterface[] = [];
 
     constructor(allNodeTypes: ExpressionMetricMapping[]) {
-        for (const nodeType of allNodeTypes) {
-            if (nodeType.metrics.includes(this.getName())) {
-                if (nodeType.expression === "type_alias_declaration") {
-                    this.statementsSuperSet.push(
-                        new SimpleLanguageSpecificQueryStatement(
-                            "(type_alias_declaration value: (object_type))",
-                            nodeType.languages,
-                            nodeType.activated_for_languages,
-                        ),
-                    );
-                } else {
-                    this.statementsSuperSet.push(
-                        new ExpressionQueryStatement(
-                            nodeType.expression,
-                            nodeType.languages,
-                            nodeType.activated_for_languages,
-                        ),
-                    );
-                }
-            }
-        }
+        this.statementsSuperSet = getQueryStatements(allNodeTypes, this.getName());
+        this.addQueriesForJS_TS_TSX();
```

Comment:
As @ce-bo did not want me to call the other function `addQueriesForC_C++`, this should be renamed, too.

```suggestion
        this.addQueriesForJsAndTsAndTsx();
```